src/.vscode/tasks.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
Auto generated file. Please, remove. It's already added to git ignore. Wondering, why it showed up here.

Reply:
It contains common tasks for the project used by VS Code. If removed, it needs to be re-created each time someone wants to do something like "build" the project. If we do want to omit it, we may want to update the ignore file and it didn't catch it when i checked it into the project.